content/build/ci/github-actions/cache.md

```diff
@@ -137,83 +137,74 @@ jobs:
 
 BuildKit doesn't preserve cache mounts in the GitHub Actions cache by default.
 If you wish to put your cache mounts into GitHub Actions cache and reuse it
-between builds, you can use a workaround provided by
+between builds, you can use this action provided by
 [`reproducible-containers/buildkit-cache-dance`](https://github.com/reproducible-containers/buildkit-cache-dance).
 
 This GitHub Action creates temporary containers to extract and inject the
 cache mount data with your Docker build steps.
 
-The following example shows how to use this workaround with a Go project.
+The following example shows how to cache the apt packages.
 
-Example Dockerfile in `build/package/Dockerfile`
+Example `Dockerfile`:
 ```Dockerfile
-FROM golang:1.21.1-alpine as base-build
-
-WORKDIR /build
-RUN go env -w GOMODCACHE=/root/.cache/go-build
-
-COPY go.mod go.sum ./
-RUN --mount=type=cache,target=/root/.cache/go-build go mod download
-
-COPY ./src ./
-RUN --mount=type=cache,target=/root/.cache/go-build go build -o /bin/app /build/src
-...
+FROM ubuntu:22.04
+ENV DEBIAN_FRONTEND=noninteractive
+RUN \
+  --mount=type=cache,target=/var/cache/apt,sharing=locked \
+  --mount=type=cache,target=/var/lib/apt,sharing=locked \
+  rm -f /etc/apt/apt.conf.d/docker-clean && \
+  echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' >/etc/apt/apt.conf.d/keep-cache && \
+  apt-get update && \
+  apt-get install -y gcc
 ```
 
 Example CI action
 ```yaml
-name: ci
-on: push
+name: CI
```

Comment:
?

Reply:
Let's keep lowercasing. This is our pattern across GHA workflows.